src/jaxsim/mujoco/loaders.py

```diff
@@ -558,7 +558,6 @@ def convert(
         # Get the ROD model.
         rod_model = load_rod_model(
             model_description=urdf,
-            is_urdf=True,
```

Comment:
Since we assume that the model is URDF, this should fail is `urdf` is a SDF string. I'd leave this flag here in this specific case.

Reply:
Addressed in https://github.com/ami-iit/jaxsim/pull/238/commits/24e91a78a0c374826827665e72277fecfce7ee41